browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -1483,6 +1483,13 @@ L.CanvasTileLayer = L.Layer.extend({
 			app.socket._logSocket('INCOMING', textMsg);
 			this.handleInvalidateTilesMsg(textMsg);
 			return true; // filter
+		} else if (textMsg.startsWith('slidelayer:')) {
```

Comment:
Sorry - this is just not ok.
Slurping tiles is an important part of making the websocket perform acceptably; and this logic has no place here.
I would like to see your log file of all the events that have occurred, what the stack traces are, what order they happen in and the fragments of code that do the actual work in the slide-show code - to work out what order things happen in and how this can go wrong.